### ISSUE TITLE
I am new to this , help me out.

### ISSUE THREAD

**Issue by baibhavKumar1** [supershaneski/openai-whisper-api#2]
Please provide what is the procedure for making this run from 0 in layman terms.

**Comment by supershaneski** [comment]
Please check the [Installation](https://github.com/supershaneski/openai-whisper-api/#installation) section.

1. First, you need to clone the repository.

```
git clone https://github.com/supershaneski/openai-whisper-api.git
```

2. Then go inside the project directory and create `.env` file. Copy the contents of `.env.example`. Enter your own OpenAI API key in `OPENAI_APIKEY`. If you do not have OpenAI account, set `DO_NOT_USE_API` to true.

2.1. If you do not have OpenAI API key, you need to install Whisper python module.
Check the [Installing Whisper Python module](https://github.com/supershaneski/openai-whisper-api#installing-whisper-python-module) section on how to install it.

2.2. If you do not have OpenAI API key yet, check the [OpenAI developer site](https://platform.openai.com/) how to get it. Upon signing up you will be given $18 in free credit that can be used during your first 3 months.

3. While inside the project directory, install the modules.

```
npm install
```

3. Then to run the app

```
npm run dev
```

4. Open your browser and go to http://localhost:3005. The browser will prompt you that you need to Allow microphone use. Press OK.

5. Click the main button to start and begin talking. That's it.

I hope the instructions are clear and simple.
